requirements/edx/github.txt

```diff
@@ -79,7 +79,7 @@ git+https://github.com/edx/XBlock.git@xblock-0.4.4#egg=XBlock==0.4.4
 -e git+https://github.com/edx/django-splash.git@v0.2#egg=django-splash==0.2
 -e git+https://github.com/edx/acid-block.git@e46f9cda8a03e121a00c7e347084d142d22ebfb7#egg=acid-xblock
 -e git+https://github.com/edx/edx-ora2.git@0.2.6#egg=ora2==0.2.6
--e git+https://github.com/edx/edx-submissions.git@0.1.3#egg=edx-submissions==0.1.3
+-e git+https://github.com/edx/edx-submissions.git@efischer/clear_state#egg=edx-submissions
```

Comment:
This branch label is just for sandbox testing; I'll update it with a properly-released new `edx-submissions` version before merging to master.